src/App.tsx

```diff
@@ -20,15 +21,15 @@ import CalendarPage from "./views/Calendar/Calendar";
 import EventPage from "./views/Message/Event";
 import * as Sentry from "@sentry/react";
 import { useAuth0 } from "@auth0/auth0-react";
-import AuthRoute from "./components/Actions/AuthRoute";
 import GsuiteProtectedRoute from "./components/Actions/GsuiteRoute";
 import { jwt } from "./api/state";
 import { useRecoilState } from "recoil";
 import CustomForm from "./views/Message/CustomForm";
-import Authorize from "./components/Actions/Authorize";
+import GsuiteAuthorize from "./components/Actions/GsuiteAuthorize";
 import ProfileInjectedTypeform from "./views/ProfileInjectedTypeform";
 import Typeform from "./components/Typeform/typeform";
-
+import Auth0ProtectedRoute from "./components/Actions/Auth0Route";
+import Auth0Authorize from "./components/Actions/Auth0Authorize";
```

Comment:
sill using auth0 :((

Reply:
Not a priority to migrate from auth0 right now. It would be interesting to explore abstracting authentication to where we can switch auth0 out in the future though. What we could do in meantime is explore handling rate limiting though.

Reply:
I'm not actually sure how we did it but for the first major TrackIT debut last year where we had 100+ new people attend for a workshop series we requested a temporary increase to our rate-limit for Auth0 new sign-ups. I think there's an option in the support center to request an increase.

Ig for that its on me to have missed thinking about it for the kickoff since all users would have been considered new users.